test/dlc_tests/benchmark/ec2/pytorch/inference/test_performance_torch_inductor_inference.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2019-2020 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+
+import os
+import time
+import pytest
+from test.test_utils import (
+    CONTAINER_TESTS_PREFIX,
+    get_framework_and_version_from_tag,
+    LOGGER,
+    BENCHMARK_RESULTS_S3_BUCKET
+)
+
+SETUP_CMD = "cd $HOME &&
+             git clone --branch v2.0.0-rc3 --recursive https://github.com/pytorch/pytorch.git &&
+             git clone --recursive https://github.com/pytorch/benchmark.git &&
+             git checkout $(cat pytorch/.github/ci_commit_pins/benchmark.txt &&
+             cd $HOME/benchmark &&
+             python install.py;"
+
+@pytest.mark.parametrize("ec2_instance_type", ["p3.16xlarge"], indirect=True)
```

Comment:
Can we get coverage for all the instances we care about here? For now we are only interested in single GPU instances - p3.2xl, c5.4xl, m5.4xl, g5.4xl, g4dn.4xl, c7g.4xl, m7g.4xl